doc/api/http.md

```diff
@@ -3206,6 +3206,43 @@ server.on('request', (request, res) => {
 server.listen(8000);
 ```
 
+## `http.createSimpleServer([directory[, port[, host]]])`
+
+<!--YAML
+added: REPLACEME
+-->
+
+> Stability: 0 - Experimental
+
+* `directory` {string|URL} Root directory from which files would be serve.
+* `port` {number}
+* `host` {string}
+
+Start a TCP server listening for connections on the given `port` and `host`, and
+serve statically local files, using `directory` as the root. Please note that
+when specifying a `host` other than `localhost`, you are exposing you local file
+system to all the machines that can connect to your computer.
+
+If `host` is omitted, the server will accept connections on the
+[unspecified IPv6 address][] (`::`) when IPv6 is available, or the
+[unspecified IPv4 address][] (`0.0.0.0`) otherwise.
```

Comment:
Why not use 127.0.0.1 and the equivalent IPv6 by default?

Reply:
I copied this from https://github.com/nodejs/node/blob/e23c256594c291dd2da481512e2dc1d0e4421375/doc/api/net.md#L544-L554

I've considered using `'localhost'` as a default value, but thought maybe consistency would be preferable, happy to change that back if others prefer it.

Reply:
I'd prefer the default to be safe in this case (avoid potentially exposing a local folder to the entire internet).